### ISSUE TITLE
Blank Backend Page in Symphony 2.7.7

### ISSUE THREAD

**Issue by animaux** [symphonists/importcsv#9]
In `Symphony 2.7.7` i get a blank backend page at `/symphony/extension/importcsv/` regardless of PHP version:

    <div id="contents" role="main">
        <form action="https://ejbweimar.de/symphony/extension/importcsv/" method="post" role="form" enctype="multipart/form-data"></form>
    </div>

No errors in any log.

**Comment by animaux** [comment]
I traced back the problem: apparently it quietly chokes on sections with ampersands in their name.

**Comment by SebastianKaminski** [comment]
I've fixed the issue by adding two lines of code:
`$xsl = preg_replace('/&(?!#?[a-z0-9]+;)/', '&amp;', $xsl);`
 `$xml = preg_replace('/&(?!#?[a-z0-9]+;)/', '&amp;', $xml);`

in `class.xsltprocess.php` file just above line 130. I hope it helps.

**Comment by animaux** [comment]
Great, can you send a PR?

**Comment by nitriques** [comment]
I do not think the fix in needed right in the core tho... I think we are missing `General::sanitize()` calls in this extension.

**Comment by SebastianKaminski** [comment]
I'm afraid that fix posted above temporary fix the extension but other Symphony functionalities are not working properly.

Edit:

I have changed line 57:

`$sectionsNode->appendChild(new XMLElement('section', $section->get('name'), array('id' => $section->get('id'))));`
to
`$sectionsNode->appendChild(new XMLElement('section', General::sanitize($section->get('name')), array('id' => $section->get('id'))));`

Issue was caused by "&" included in section name. All seems to be working now, but I need to do further tests.

@nitriques Thanks for `General::sanitize()` tip.
